examples/lists_example.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Test for list structures in PyLaTeX.
+# More info @ http://en.wikibooks.org/wiki/LaTeX/List_Structures
+
+from pylatex import Document, Section, Itemize, Enumerate, Description
+
+
+doc = Document()
+
+# create a bulleted "itemize" list like the below:
+# \begin{itemize}
+#   \item The first item
+#   \item The second item
+#   \item The third etc \ldots
+# \end{itemize}
+
+with doc.create(Section('"Itemize" list')):
+    with doc.create(Itemize()) as enum:
```

Comment:
It's a bit confusing that everything is called enum, when its not all an Enumerate class.